### ISSUE TITLE
Wrong --libs in pkg-config for fimex in conda

### ISSUE THREAD

**Issue by heikoklein** [metno/fimex#36]
Hi,
the pkg-config for fimex in conda gives
```
$ pkg-config --libs fimex
-L/modules/rhel8/user-apps/fou-modules/conda/atom-fimex/2022-10-21/modules/rhel8/user-apps/fou-modules/conda/atom-fimex/2022-10-21/lib -lfimex 

```
i.e. the -L/path is written twice giving -L/path/path
which follows from the libdir in fimex.pc file
```
prefix=/modules/rhel8/user-apps/fou-modules/SnapPy/TEST_HK
libdir=${prefix}//modules/rhel8/user-apps/fou-modules/SnapPy/TEST_HK/lib
includedir=${prefix}/include
```

Tested with both fimex-1.8.1 and 1.9.4.

**Comment by alexander-buerger-met-no** [comment]
I am not sure where this comes from. If I install `fimex` and `pkg-config` in a fresh conda environment, inside this environment I get:
```
$ pkg-config --libs fimex
-lfimex-1.9
```
This looks correct to me. The lib search path inside the conda environment includes the path where `libfimex.so` is actually installed.

**Comment by alexander-buerger-met-no** [comment]
Sorry, I checked with a bad conda environment.

The problem comes from some cmake install variables being relative, others absolute.

**Comment by alexander-buerger-met-no** [comment]
Should be solved with commit 0d1159221ac92a65ce59c3ec5eb968344da21474.
